app/controllers/planes_controller.rb

```diff
@@ -1,13 +1,31 @@
 class PlanesController < ApplicationController
   def index
+    @planes = Plane.all
   end
 
   def show
+    @plane = Plane.find(params[:id])
   end
 
   def new
+    @owner = current_user.id
```

Comment:
should this also be @owner = current_user

Reply:
Thanks, changing now